src/contract_wrappers/tokens/security_token_wrapper.ts

```diff
@@ -1108,7 +1116,7 @@ export default class SecurityTokenWrapper extends ERC20TokenWrapper {
       params.indexFilterValues,
       SecurityToken.abi,
       params.callback,
-      params.isVerbose,
+      !_.isUndefined(params.isVerbose),
```

Comment:
```suggestion
      params.isVerbose,
```
And lodash can be removed.